Sources/SwiftGenStringsCore/Output Formatting/ErrorFormatter.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+public struct ErrorFormatter {
+
+    public init() {}
+
+    public func writeFormattedError(_ error: Error) {
+        let formattedError = "[ERROR] \(error.localizedDescription)\n".formattedStringWithColor(.red)
```

Comment:
just to be double sure - please test this with the internal repository and the `regenerateEnglishStrings` method as we're changing the output format

Reply:
Only the error output has been changed, which results in no output (that is intended to be used) to be written. I tested it though and it worked just fine :)